crates/fj-viewer/src/gui.rs

```diff
@@ -24,12 +24,6 @@ use fj_math::{Aabb, Scalar};
 
 use crate::graphics::DrawConfig;
 
-/// Event that are passed between the event_loop and gui
-pub enum GuiEvent {
-    AskModel,
-    LoadModel(PathBuf),
-}
```

Comment:
Removing `GuiEvent` outright is perfectly fine, but just for the record, I think it would have been slightly nicer to keep a `struct AskModel;` and a `struct LoadModel(PathBuf);` around, to make those `Sender`/`Receiver` types below a bit more descriptive (`Sender<AskModel>` instead of `Sender<()>`, for example).

But this is only a minor thing! Just mentioning it for future reference.